lib/pirate.rb

```diff
@@ -0,0 +1,14 @@
+# Path: mythical-creatures/lib/pirate
+
+# Practice git and ruby testing with Stemmle
+class Pirate
+  attr_reader :name, :job, :act, :booty
+
+  def initialize(name, job = 'Scallywag', act = 0, booty = 0)
+    @name = name
+    @job = job
+    @act = act
+    @booty = booty
+  end
```

Comment:
@Taamir-Ransome couple things here. 
1. coder empathy: the term `act` here would leave too much up to interpretation. `name`, `booty`,  and `job` definitely easy to know whats going on. If a pirate can commit heinous acts, then lets make that explicit
2. object instantiation: I don't really see a good reason to have `act`, and `booty` as method arguments

Reply:
@Taamir-Ransome also consider the use of a keyword arg for job to increase readability...
example:
```
taamir = Pirate.new('T', job: 'new dev')

Reply:
@Taamir-Ransome before I merge this lets see if you can make some changes, amend your commits, and force push...
changes:
1. move `act` and `booty` out of args
2. rename `act` to `heinous_acts`